arm-network/2017-06-01/swagger/applicationGateway.json

```diff
@@ -390,7 +390,7 @@
         "tags": [
           "ApplicationGateways"
         ],
-        "operationId": "ApplicationGateways_ListAvailableSslOptions",
+        "operationId": "ApplicationGateways_AvailableSslOptions",
```

Comment:
From the internal email thread with Laurent who maintains Py SDK, it was implied that backward compat with 2017-03-01 was indeed a good thing, if not a requirement. With that in mind and the existing state of the spec, plus that this change adds no more errors/warns, I'm okay with keeping the existing bad naming for now.